db/seeds.rb

```diff
@@ -182,5 +199,60 @@
   name: 'Admin',
   email: 'admin@wnb.rb',
   password: 'password',
-  password_confirmation: 'password'
+  password_confirmation: 'password',
+)
+
+p 'Create 4 jobs on the job board'
```

Comment:
Actual changes start here -- add 4 new jobs to the seeds to render in our job board.